src/entity.rs

```diff
@@ -12,7 +12,7 @@ pub const SMALL_VECTOR_SIZE: usize = 8;
 /// Marker for elements in a vector. Let's say that we have `vec![1, 2, 3, 4]`
 /// and `LengthAndOffset { length: 2, offset : 1 }`. Offset points to the second element in the vector
 /// and length tell us how many elements we should take (in that case 2 elements: 2 and 3).
-#[derive(Clone, Copy)]
+#[derive(Debug, Clone)]
```

Comment:
copy was fine here; it's 8 bytes, so we can copy it heaply

Reply:
Copy is unused and not required anywhere. Do we still want to keep it?